public/docs/dart/latest/tutorial/toh-pt6.jade

```diff
@@ -1 +1,139 @@
-!= partial("../../../_includes/_ts-temp")
+extends ../../../ts/latest/tutorial/toh-pt6.jade
+
+block includes
+  include ../_util-fns
+  - var _Http = 'BrowserClient';
+  - var _Angular_Http = 'Dart <code>BrowserClient</code>'
+  - var _httpUrl = 'https://pub.dartlang.org/packages/http'
+  - var _Angular_http_library = 'Dart <a href="' + _httpUrl + '"><b>http</b></a> library'
+  - var _HTTP_PROVIDERS = 'BrowserClient'
+  - var _JSON_stringify = 'JSON.encode'
+
+block start-server-and-watch
+  :marked
+    ### Keep the app compiling and running
+    Open a terminal/console window.
+    Start the Dart compiler, watch for changes, and start our server by entering the command:
+
+  code-example(language="bash").
+      pub serve
+ 
+block http-library
+  :marked
+    We'll be using the !{_Angular_Http} to communicate with a server.
+    This class is exported by the !{_Angular_http_library}.
```

Comment:
Yes, I saw that too. :) I wasted a few minutes of my life thinking about what that meant, but once I saw that we couldn't get BrowserClient by importing http, muddling the library and package seemed wrong.